xla/stream_executor/device_description.cc

```diff
@@ -140,9 +155,9 @@ bool ThreadDimOk(const DeviceDescription &device_description,
   return ok;
 }
 
-void CalculateDimensionality(const DeviceDescription &device_description,
-                             int64_t element_count, int64_t *threads_per_block,
-                             int64_t *block_count) {
+void CalculateDimensionality(const DeviceDescription& device_description,
```

Comment:
Please don't do unrelated style fixes in the PR -- it pollutes the "blame" history unnecessarily. Just do the changes you need. That said, if you're changing certain lines because you need them for your change, feel free to fix the style on those lines.